plugins/azure_sentinel/help.md

```diff
@@ -0,0 +1,613 @@
+        "additionalData":{
+          "ETag":"\"3b00acab-0000-0d00-0000-5f15e4ed0000\"",
+          "EntityId":"afbd324f-6c48-459c-8710-8d1e1cd03812"
+        },
+        "friendlyName":"SecurityEvent - 868f40f4698d"
+      }
+    }
+  ]
+}
+```
+### Triggers
+
+_This plugin does not contain any triggers._
+
+### Custom Output Types
+
+#### CreatedByType
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Application|string|False|Application|
+|Key|string|False|Description|
+|Managed Indetity|string|False|Managed Identity|
+|User|string|False|User|
+
+#### Incident
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|etag|string|False|None|
+|Full Incident ID|string|False|Full incident ID|
+|Incident ID|string|False|Incident name - short id|
+|Incident properties|IncidentProperties|False|Incident properties object.|
+|type|string|False|None|
+
+#### IncidentAdditionalData
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|List of product names of alerts in the incident|[]string|False|List of product names of alerts in the incident|
+|The number of alerts in the incident|integer|False|The number of alerts in the incident|
+|The number of bookmarks in the incident|integer|False|The number of bookmarks in the incident|
+|The number of comments in the incident|integer|False|The number of comments in the incident|
+|The tactics associated with incident|[]string|False|The tactics associated with incident|
+
+#### IncidentLabel
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|The name of the label|string|False|The name of the label|
+|The type of label|string|False|The type of label.|
+
+#### IncidentOwnerInfo
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|The name of the user the incident is assigned to.|string|False|The name of the user the incident is assigned to.|
+|The email of the user the incident is assigned to.|string|False|The email of the user the incident is assigned to.|
+|The object id of the user the incident is assigned to.|string|False|The object id of the user the incident is assigned to.|
+|The user principal name of the user the incident is assigned to.|string|False|The user principal name of the user the incident is assigned to.|
+
+#### IncidentProperties
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|Additional data on the incident|IncidentAdditionalData|False|Additional data on the incident|
+|The reason the incident was closed|string|False|The reason the incident was closed|
+|Describes the reason the incident was closed|string|False|Describes the reason the incident was closed|
```

Comment:
Are these really the property names?

Reply:
It seems there is a bug in the generation of `help.md`